cdap-common/src/main/java/co/cask/cdap/common/security/AuthEnforceRewriter.java

```diff
@@ -439,24 +506,43 @@ public void visitEnd() {
    * @return a new {@link Map} where the key is the parameter name given in the annotation and value is the
    * position of the parameter.
    */
-  private static Map<String, Integer> processParameterAnnotationNode(Map<Integer, AnnotationNode>
+  private static Map<String, Integer> processParameterAnnotationNode(Map<Integer, ParameterDetail>
```

Comment:
Rename the method appropriately (e.g. `processParameterDetails` or `getAnnotationValuePosition`).